### PR TITLE
TrailingSpacesFixer - stop changing non-whitespacy tokens

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by keradus** [comment]
Ref #923

**Comment by keradus** [comment]
Ref https://github.com/SpacePossum/PHP-CS-Fixer/pull/2

**Comment by SpacePossum** [comment]
All tests are green, your new fixer works! :D  @keradus

**Comment by keradus** [comment]
Any opinions ?

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by keradus** [comment]
Thank you @SamBurns-awin

File: Symfony/CS/Fixer/PSR2/TrailingSpacesFixer.php [hunk 1]
```diff
@@ -25,8 +29,32 @@ class TrailingSpacesFixer extends AbstractFixer
...
+
+        foreach ($tokens as $index => $token) {
+            if (!$token->isWhitespace()) {
+                continue;
+            }
+
+            $lines = preg_split("/([\r\n]+)/", $token->getContent(), -1, PREG_SPLIT_DELIM_CAPTURE);
+            $linesSize = count($lines);
+
+            // fix only multiline whitespaces or singleline whitespaces at the and of file
```
Comment (stof) [inline Symfony/CS/Fixer/PSR2/TrailingSpacesFixer.php]: typo: ``end``

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/TrailingSpacesFixer.php]: Thank you, I have incorporate that tiny fix in #1332

**Pull request closed without merging**